benchmarks/shelley3pools/configuration/topology-node-1.json

```diff
@@ -1,5 +1,10 @@
 {
   "Producers": [
+    {
```

Comment:
we should build topologies from a template and point node N to N+1, N+2
target node numbers will be calculated e.g. $(((N+2) % NNODES))  (modulo number of nodes)

Reply:
I don't know what happened to these files, but they come out okay when you run `./topology-generator.sh`. I'll just delete them from this PR.